cmd/krel/cmd/gcbmgr.go

```diff
@@ -271,7 +271,7 @@ func RunGcbmgr(opts *GcbmgrOptions) error {
 		jobType = "stage"
 	case opts.Release:
 		jobType = "release"
-		buildOpts.DiskSize = "100"
+		buildOpts.DiskSize = "500"
```

Comment:
Can we do the same for stage? I mean we build the binaries in `stage` so I guess we already need the increased disk size there…

Reply:
stage was using the default value, which was 300Gb, set the default to 500 and remove the custom setting for release step, so now all use 500

I prefer to use the config in the cloudbuild, with that we avoid code changes.
we can check the cloudbuild.yaml and if is not defined we can use some default in the code (two cents)